src/LEGO.AsyncAPI/Writers/SpecialCharacterStringExtensions.cs

```diff
@@ -184,10 +184,10 @@ internal static string GetYamlCompatibleString(this string input)
             }
 
             // If string can be mistaken as a number, a boolean, or a timestamp,
-            // wrap it in quote to indicate that this is indeed a string, not a number, a boolean, or a timestamp
-            if (decimal.TryParse(input, NumberStyles.Float, CultureInfo.InvariantCulture, out var _) ||
+            // wrap it in quot number, a boolean, or a timestamp
+            if (decimal.TryParse(input, NumberStyles.Float, Configuration.CultureInfo, out var _) ||
                 bool.TryParse(input, out var _) ||
-                DateTime.TryParse(input, out var _))
+                DateTime.TryParseExact(input, Configuration.DateTimeFormat, Configuration.CultureInfo, DateTimeStyles.RoundtripKind, out var _))
```

Comment:
Did you add tests for this particular change? In the 'old' version of the change, you had a new test class specifically for testing this extension.

Reply:
Added it back :)